src/plugins/exitscript/exitscript.js

```diff
@@ -0,0 +1,141 @@
+/**
+* @title WET-BOEW Exit script plugin
+* @overview Plugin redirects users to non secure site
+* @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+* @author @ipaksc
+*/
+( function( $, window, wb ) {
+"use strict";
+var componentName = "wb-exitscript",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+	exiturlparam = componentName + "-urlparam",
+	moDalId = componentName + "-modal",
+	i18n,
+	i18nDict = {
+		en: {
+			"msgboxHeader": "Warning",
+			"exitMsg": "You are about to leave a secure site, do you wish to continue?",
+			"targetWarning": "The link will open in a new browser window.",
+			"yesBtn": "Yes",
+			"cancelBtn": "Cancel"
+
+		},
+		fr: {
+			"msgboxHeader": "Avertissement",
+			"exitMsg": "Vous êtes sur le point de quitter un site sécurisé. Voulez-vous continuer?",
+			"targetWarning": "Le lien s'ouvrira dans une nouvelle fenêtre de navigateur",
+			"yesBtn": "Oui",
+			"cancelBtn": "Annuler"
+
+		}
+	},
+
+	/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector ),
+			settings,
+			originalURL = decodeURIComponent( decodeURIComponent( String( wb.getUrlParts( window.location.href ).params.exturl ) ) ),
+			$elm;
+		if ( elm ) {
+			$elm = $( elm );
+			settings = $.extend(
+				true,
+				window[ componentName ],
+				wb.getData( $elm, componentName )
+
+			);
+
+			$elm.data( componentName, settings );
+
+			if ( settings.exitURL ) {
+				$( this ).attr( "href", settings.exitURL + "?exturl=" +  encodeURIComponent( this.getAttribute( "href" ) ) );
+			}
+
+			if ( !i18n ) {
+				i18n = i18nDict[ wb.lang || "en" ];
+			}
+
+			if ( $elm.hasClass( exiturlparam ) ) {
+				this.outerHTML = "<a href='" + originalURL + "'>" + originalURL + "</a>";
+			}
+			wb.ready( $elm, componentName );
+		}
+	};
+
+$document.on( "click", selector, function( event ) {
+
+	var elm = event.currentTarget,
+		$elm = $( elm ),
+		wrapper,
+		targetAttribute,
+		moDal = document.createDocumentFragment(),
+		tpl = document.createElement( "div" ),
+		settings =  $elm.data( componentName ),
+		msgboxHeader = settings.msgboxHeader || i18n.msgboxHeader,
+		yesBtn = settings.yesBtn || i18n.yesBtn,
+		cancelBtn = settings.cancelBtn || i18n.cancelBtn,
+		exitMsg = settings.exitMsg || i18n.exitMsg;
+
+	if ( !settings.exitURL ) {
+
+		event.preventDefault();
+	}
+
+	if ( this.getAttribute( "target" ) !== null ) {
+		targetAttribute = "target='" + this.getAttribute( "target" ) + "'";
+		exitMsg = exitMsg  + " " + i18n.targetWarning;
+	} else {
+		targetAttribute = "";
```

Comment:
Leave the default in the variable declaration, like at line 75. This else condition is not needed